modules/openapi-generator/src/main/resources/python/python-experimental/signing.mustache

```diff
@@ -0,0 +1,315 @@
+# coding: utf-8
+{{>partial_header}}
+from __future__ import absolute_import
+
+from datetime import timedelta
+from six.moves.urllib.parse import urlencode, urlparse
+import pem
+from Crypto.PublicKey import RSA, ECC
+from Crypto.Signature import PKCS1_v1_5, pss, DSS
+from Crypto.Hash import SHA256, SHA512
+from base64 import b64encode
+from email.utils import formatdate
+
+from {{packageName}}.configuration import HttpSigningConfiguration
```

Comment:
I just created an integration branch that takes 3 PRs together. I won't merge it, but at least it will allow me to build and run the unit tests. I will fix this issue and any other issue that comes up.